src/chocolatey/infrastructure.app/registration/ChocolateyRegistrationModule.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright © 2017 - 2022 Chocolatey Software, Inc
+// Copyright © 2017 - 2022 Chocolatey Software, Inc

```

Comment:
```suggestion
// Copyright © 2017 - 2023 Chocolatey Software, Inc

```

Noticed that this was changed in another file, but not here. Was there a reason for that?

Reply:
Just didn't catch it, I'll get that updated.